pkg/accounting/accounting.go

```diff
@@ -302,6 +305,8 @@ func (a *Accounting) PrepareCredit(ctx context.Context, peer swarm.Address, pric
 }
 
 func (c *creditAction) Apply() error {
+	loggerV1 := c.accounting.logger.V(1).Register()
```

Comment:
we can just name it logger as there are no other versions

Reply:
Good point, I wanted to be explicit so you know you are logging at level `v=1`. However, I'm not strict about this.